tests/unit_test/callback/pyscript_handler_test.py

```
@@ -1484,9 +1478,9 @@ def test_add_schedule_job_success(monkeypatch):
 
     monkeypatch.setattr(ActionUtility, "execute_http_request", lambda url, method: {"success": True})
 
-    monkeypatch.setattr(CallbackUtility, "datetime_to_utc_timestamp", lambda dt: 1234567890)
+    monkeypatch.setattr(CallbackScriptUtility, "datetime_to_utc_timestamp", lambda dt: 1234567890)
 
-    result = CallbackUtility.add_schedule_job(schedule_action, date_time, data, tz, _id=_id, bot=bot, kwargs=kwargs_in)
+    result = CallbackScriptUtility.add_schedule_job(schedule_action, date_time, data, tz, _id=_id, bot=bot, kwargs=kwargs_in)
```

Ruff (0.8.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)